roadmap.md

```diff
@@ -99,6 +99,20 @@ Architecture:
 - [.NET Standard](https://github.com/dotnet/standard)
 - [.NET Core Tools/CLI](https://docs.microsoft.com/en-us/dotnet/articles/core/tools/index)
 
+Version 2.0 minimum OS Support (Proposed):
+
+OS                            |Version                        |Architectures|Configurations|Notes
+------------------------------|-------------------------------|-------------|--------------|-----
+Windows Client                | 7 SP1 - 10                    | x64, x86    |              |
+Windows Server                | 2008 R2 SP1 - 2016            | x64, x86    | Full, Server Core, Nano |
+Red Hat Enterprise Linux      | 7.2                           | x64         |              | This includes Centos and Oracle Linux
+Fedora                        | 25                            | x64         |              |
+Debian                        | 8.2                           | x64         |              |
+Ubuntu                        | 14.04, 16.04                  | x64         |              | This includes Linux Mint 17, 18
```

Comment:
How about Ubuntu arm? Could you please let me know the reason why you didn't add arm architecture here?